.github/workflows/test.yaml

```diff
@@ -5,7 +5,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go: [ 1.18, 1.19 ]
+        go: [ 1.19, 1.20 ]
```

Comment:
you'll have to quote `"1.20"`, easiest to just quote both to avoid the mistake in future (1.20 is 1.2 not 1.20)

Reply:
:man_facepalming: Yep.

Reply:
Made the same mistake on other projects hehe